kite-api/src/utils/mongo.util.js

```diff
@@ -4,7 +4,7 @@ const bcrypt = require('bcrypt');
 const { saltRounds } = require('../../config/config');
 
 const dbConnect = () => {
-  mongoose.connect(`mongodb://kitemongo:27017/kite`, { useNewUrlParser: true });
+  mongoose.connect(`mongodb://kitemongo:27017/kite`, { useNewUrlParser: true, useFindAndModify: false });
```

Comment:
Nice 😎😎😎

Reply:
Thanks, removes a Mongo deprecation warning for us.